src/APICalls.js

```diff
@@ -7,5 +7,17 @@ export const getSingleRecipe = (id) => {
   return fetch(`https://www.thecocktaildb.com/api/json/v2/9973533/lookup.php?i=${id}`)
 }
 
-//https://www.thecocktaildb.com/api/json/v2/9973533/search.php?s=
-// ^^ big api with 627 drinks. I used the 'latest' list so it wasn't so huge.
+export const getVodkaRecipes = () => {
```

Comment:
Maybe we could shorten these API calls to be more dynamic 
For example:
```
export const getIngredientRecipes = (ing) => {
  return fetch(`https://www.thecocktaildb.com/api/json/v2/9973533/filter.php?i=${ing}`)
  .then(response => response.json())
}
```

Then just pass in the ingredient rather than the individual API call.